internal/proto/volumegroup.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+package proto;
+
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/proto";
```

Comment:
Can you check again ? 
I don't think this matches csi-addons spec or cephcsi implementation
https://github.com/csi-addons/spec/tree/main/volumegroup#csi-addons-operation-volumegroup
https://github.com/ceph/ceph-csi/blob/devel/internal/csi-addons/rbd/volumegroup.go

Let's implement everything here too and ignore information that's not needed right now at controller level.

Reply:
Updated it now to include everything as per the sepc defined in the csi-addons. Please check now.